bot/src/wrapper/_communication.py

```diff
@@ -208,3 +211,60 @@ async def caller(session: aiohttp.ClientSession) -> ShopBuyOutput:
                 raise UnknownNetworkError(message)
 
         return await make_request(session, caller)
+
+
+class UserRawAPI:
+    """A bundle of API available to use for user endpoint."""
+
+    @staticmethod
+    async def get_user(session: aiohttp.ClientSession, user_id: int) -> UserIdGetOutput:
+        """Get the User information by user id through direct HTTP request."""
+
+        async def caller(session: aiohttp.ClientSession) -> UserIdGetOutput:
+            async with session.get(f"user/{user_id}") as resp:
+                if resp.ok:
+                    return await resp.json()
+                if resp.status == Status.NOT_FOUND:
+                    message = f"User with user id {user_id} cannot be found"
+                    raise DoNotExistError(message)
+                message = f"Undefined behaviour bot.src.wrapper.UserRawAPI.get_user, Status received {resp.status}"
+                raise UnknownNetworkError(message)
+
+        return await make_request(session, caller)
+
+    @staticmethod
+    async def create_user(session: aiohttp.ClientSession, user_id: int) -> None:
+        """Create the user by user id through a direct HTTP request."""
+
+        async def caller(session: aiohttp.ClientSession) -> None:
+            async with session.get(f"user/{user_id}") as resp:
+                if resp.ok:
+                    return
+                if resp.status == Status.CONFLICT:
+                    message = f"User with user id {user_id} have been register already"
+                    raise AlreadyExistError(message)
+                message = f"Undefined behaviour bot.src.wrapper.UserRawAPI.create_user, Status received {resp.status}"
+                raise UnknownNetworkError(message)
+
+        return await make_request(session, caller)
+
+    @staticmethod
+    async def update_user_experience(
+        session: aiohttp.ClientSession, user_id: int, src: UserIdExperiencePatchInput
+    ) -> UserIdExperiencePatchOutput:
+        """Get the user experience by user id through a direct HTTP request."""
+
+        async def caller(session: aiohttp.ClientSession) -> UserIdExperiencePatchOutput:
+            async with session.patch(f"user/{user_id}/experience", json=src) as resp:
```

Comment:
Technically the v2 doc states a POST request however PATCH makes more sense so my bad :)
I'll update the v3 doc

Reply:
It would be in a separate branch that base on `wrapperv2` call `wrapperv3`

Reply:
Oh, oops, just notice I also implemented this wrong